.github/workflows/linter.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         env:
           VALIDATE_ALL_CODEBASE: false
           VALIDATE_MARKDOWN: false
-          GITHUB_TOKEN: ${{ secrets.SECRET_GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Since develop branch has not been updated with this change and feature branch requires this github_token instead of secret_github_action to run this linter and this branch has also been checked out from develop. Therefore we need this change in current scenario.